### PR TITLE
Read me

### PR DESCRIPTION
**Pull request by AxelVivien25**

### CODE REVIEW

**Comment by t-henri** [comment]
Can you do this PR from a branch created from master ? Here there are code changes from your previous PR

File: README.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+## Installation
+
+In a python environment :
+
+```bash
+$ git clone git clone https://github.com/criteo/mlflow-elasticsearchstore.git
+$ cd mlflow-elasticsearch
+$ pip install .
+```
```
Comment (t-henri) [inline README.md:13]: You can specify that this is for development, for a common usage we will publish a package on pypi so `pip install mlflow-elasticsearchstore` should be sufficient (do not put it for now since we haven't published the package yet)

File: README.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+## Launch
```
Comment (t-henri) [inline README.md:15]: I wouldn't name this part Launch since we don't actually launch the store, it's more a How to

File: README.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
+# mlflow-elasticsearchstore
+
+Mlflow plugin to use Elaticsearch as Tracking Store in Mlflow. To use this plugin you need a running instance of Elasticsearch 6.8.
```
Comment (t-henri) [inline README.md:3]: I would rephrase this, the tracking service is MLflow, and elastic search is a backend you can use with this plugin. Also does it need to be Elasticsearch 6.8 ? Doesn't it work with 6.X ?

Comment (AxelVivien25) [reply README.md:3]: Haven't tested it but it should work
